cts_runner/test.lst

```diff
@@ -87,6 +87,12 @@ webgpu:api,validation,encoding,queries,general:occlusion_query,query_index:*
 webgpu:shader,validation,extension,dual_source_blending:blend_src_syntax_validation:*
 webgpu:api,validation,image_copy,buffer_texture_copies:depth_stencil_format,copy_usage_and_aspect:*
 webgpu:api,validation,image_copy,buffer_texture_copies:depth_stencil_format,copy_buffer_size:*
+// `vulkan` failure: https://github.com/gfx-rs/wgpu/issues/????
+fails-if(vulkan) webgpu:api,validation,queue,buffer_mapped:writeBuffer:*
```

Comment:
Just curious: is the vulkan issue related?

Reply:
I'm not sure, since I haven't really dived into it. It was failing before, and continues to fail now (see also 2c075db1), so I figured we might as well take the win for other platforms.

Reply:
I filed <https://github.com/gfx-rs/wgpu/issues/8479> for follow-up.